packages/sdk-codegen/src/kotlin.gen.spec.ts

```diff
@@ -59,29 +59,26 @@ describe('Kotlin generator', () => {
  * Type of permission: "view" or "edit" Valid values are: "view", "edit".
  */
 enum class PermissionType : Serializable {
-  view,
-  edit
+    view,
+    edit
 }`
       expect(actual).toEqual(expected)
     })
+
     it('special needs', () => {
       const type = apiTestModel.types.HyphenType
       const actual = gen.declareType('', type)
-      const expected = `data class HyphenType (
-  /**
-   * A normal variable name (read-only)
-   */
-  var project_name: String? = null,
-  /**
-   * A hyphenated property name (read-only)
-   */
-  @JsonProperty("project-digest")
-  var project_digest: String? = null,
-  /**
-   * A spaced out property name (read-only)
-   */
-  @JsonProperty("computation time")
-  var computation_time: Float? = null
+      const expected = `/**
```

Comment:
Oh, what command should I have been running to catch these changes?

Reply:
If I removed the Looker.3.1.oas.json or Looker.4.0.oas.json that was a commit mistake.

`yarn jest packages/sdk-codegen/` would run those tests. Still need to work on our test suite scripts.

Reply:
I'm not seeing removal of the needed spec files. That folder is git ignored by default. When API Explorer no longer needs to bundle the OpenAPI spec files @josephaxisa or I will again remove the `*.oas.json` files from the repo.

Reply:
@jkaster, regarding @kalenp's comment about removed files (and hence the -33k loc) you need to:

```
git reset origin/master spec/Looker.3.1.oas.json spec/Looker.4.0.oas.json
git checkout origin/master spec/Looker.3.1.oas.json spec/Looker.4.0.oas.json
git commit -am 'undeleting oas spec files
```